### DISCUSSION TITLE
Achte Woche

### DISCUSSION THREAD

**Discussion by davidlzt** [MelonigeMelone/xCalendar#22] [Blogkanal]
# Woche 8

##ASR

### Anforderungen an Qualitätsmerkmale klären

Architektur:
Frontend: Angular
Backend: GraphQL

o Qualitätsszenarien: "Quelle-Auslöser-Artefakt-Umgebung-Antwort-
Antwortmaß“
o Verwenden Sie einen Qualitätsbaum als Leitfaden für die Diskussion und ordnen Sie jedem Qualitätsmerkmal die Qualitätsszenarien zu
### Diskutieren Sie, welche potenziellen Strategien für Ihr Projekt geeignet sind
o Konzentrieren Sie sich auf die 1-2 Qualitätsszenarien mit hohen Prioritäten.
o Nutzen Sie Strategiebäume (siehe Tafelanschrieb oder1) und die Checklisten2 zur Unterstützung Ihrer Diskussion.
### Architekturentscheidungen und Entwurfsmuster treffen und dokumentieren
o Was sind die Entscheidungen und Begründungen dafür?

## Erstellen eines Qualitätsbaums

Diese Woche haben wir einen Qualitätsbaum mit vielen Qualitätsmerkmale und 5 Szenarien (mehr gratis Elemente erlaubt die Seite nicht) erstellt. 
![ScreenshotQualitätstree](https://github.com/MelonigeMelone/xCalendar/assets/113108281/5bc9e7f8-040b-477a-ac1a-07edbac29873)

## TBD

**Comment by JanWilfert** [comment]
Hallo zusammen, 
euer Qualitätsbaum sieht gut aus und ist auch sehr ausführlich. 
Wo findet man denn eure ASR?

**Comment by Charlotte0999** [comment]
Wir finden auch, dass euer Baum sehr ausführlich geworden ist und 5 Szenarien für den Anfang mehr als ausreichen.
